Faultify.TestRunner/ProjectDuplication/TestProjectDuplicator.cs

```diff
@@ -12,15 +12,27 @@ public class TestProjectDuplicator
     {
         private readonly string _testDirectory;
 
+        private DirectoryInfo newDirInfo;
+
+        private List<string> allFiles;
+
+        private IProjectInfo projectInfo;
+
         public TestProjectDuplicator(string testDirectory)
         {
             _testDirectory = testDirectory;
         }
 
-        public List<TestProjectDuplication> MakeInitialCopies(IProjectInfo testProject, int count)
+        /// <summary>
+        ///  Create the very first duplication of the project that will be used for coverage calculations
```

Comment:
[](url)
```suggestion
        ///  Creates the very first duplication of the project that will be used for coverage calculations.
```

I usually use a plural beginning for public methods. Just for the sake of consistency.